packages/core/src/interface/graph.ts

```diff
@@ -528,13 +528,15 @@ export interface IAbstractGraph extends EventEmitter {
    * 收起指定的 Combo
    * @param comboId combo ID 或 combo 实例
    */
-  collapseCombo: (combo: string | ICombo, stack?: boolean) => void;
+  // collapseCombo: (combo: string | ICombo, stack?: boolean) => void;
+  collapseCombo: (combo: string | ICombo, stack?: boolean, opts?: {inheritLabelWithDir: boolean, showCount: boolean}) => void;
```

Comment:
can we rename 
inheritLabelWithDir -> inheritLabel

and explain in the jsdoc above what criteria it is applying when set to true